### PR TITLE
Change ProductPrice Installments logic

### PR DESCRIPTION
**Pull request by estacioneto**
#### What is the purpose of this pull request?
Add some Installments logic to the ProductPrice component so it can decide the best installment to show.

#### What problem is this solving?
The Installment logic was not defined yet.

#### How should this be manually tested?
[Access the workspace](https://installments--storecomponents.myvtex.com/) and see it inside the ProductSummary and ProductDetails

#### Screenshots or example usage
image
image

#### Types of changes
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [x] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/ProductPrice/index.js [hunk 1]
```diff
@@ -15,13 +16,24 @@ class Price extends Component {
...
       showLabels,
       showSavings,
       intl: { formatNumber },
     } = this.props
 
+    const noInterestRateInstallments = installments.filter(
+      installment => !installment.InterestRate
+    )
+
+    const installment = (isEmpty(noInterestRateInstallments)
```
Comment (lucasecdb) [inline react/components/ProductPrice/index.js]: just something minor, but could you add a comment briefly explaining this logic?

Comment (estacioneto) [reply react/components/ProductPrice/index.js]: Ok

File: react/components/ProductPrice/index.js [hunk 2]
```diff
@@ -128,10 +148,21 @@ Price.propTypes = {
   showInstallments: PropTypes.bool.isRequired,
   /** Set visibility of savings */
   showSavings: PropTypes.bool,
-  /** Available number of installments */
-  installments: PropTypes.number,
-  /** Single installment price */
-  installmentPrice: PropTypes.number,
+  /** Available installments */
```
Comment (brunojdo) [inline react/components/ProductPrice/index.js]: README

File: react/components/ProductPrice/index.js [hunk 3]
```diff
@@ -128,10 +148,21 @@ Price.propTypes = {
   showInstallments: PropTypes.bool.isRequired,
```
Comment (brunojdo) [inline react/components/ProductPrice/index.js]: I think that this propTypes were big enough to be extracted to another js. `propTypes.js `

File: react/components/ProductPrice/README.md [hunk 4]
```diff
@@ -1,13 +1,17 @@
...
 ```
 
 ## Usage
-You can use it in your code like a React component with the jsx tag: `<ProductPrice />`. 
+
+You can use it in your code like a React component with the jsx tag: `<ProductPrice />`.
+
 ```html
 <ProductPrice
     listPrice={commertialOffer.ListPrice}
```
Comment (brunojdo) [inline react/components/ProductPrice/README.md]: This sample change too, right?

Comment (estacioneto) [reply react/components/ProductPrice/README.md]: Yep

**Pull request merged**